plugin/trino-kafka/pom.xml

```diff
@@ -17,6 +17,17 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
```

Comment:
This is already added in root pom.xml with snapshot disabled

Reply:
ok, got it